templates/repo/actions/runs_list.tmpl

```diff
@@ -16,10 +16,12 @@
 				</a>
 				<div class="flex-item-body">
 					<b>{{if not $.CurWorkflow}}{{.WorkflowID}} {{end}}#{{.Index}}</b>
-					: {{$.locale.Tr "actions.runs.commit"}}
+					: {{if .ScheduleID}}{{$.locale.Tr "actions.runs.scheduled"}}
+					{{else}}{{$.locale.Tr "actions.runs.commit"}}
 					<a href="{{$.RepoLink}}/commit/{{.CommitSHA}}">{{ShortSha .CommitSHA}}</a>
 					{{$.locale.Tr "actions.runs.pushed_by"}}
 					<a href="{{.TriggerUser.HomeLink}}">{{.TriggerUser.GetDisplayName}}</a>
+					{{end}}
```

Comment:
I think the code doesn't need to be so compact.

Reply:
Yes, let's properly indent the `if`